docs/tutorials/data_pipeline.md

```diff
@@ -98,11 +98,11 @@ For each operation, we list the related dict fields that are added/updated/remov
 
 `GlobalRotScaleTrans`
 - add: pcd_trans, pcd_rotation, pcd_scale_factor
-- update: points, *bbox3d_fields
+- update: points, bbox3d_fields
```

Comment:
I think the `*` here indicate that, `input_dict['bbox3d_fields']` itself is not changed, but `input_dict[item] for item in bbox3d_fields` will be changed. For example [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/pipelines/transforms_3d.py#L607-L608) in `GlobalRotScaleTrans`, it changes all the boxes, where the boxes key in the dict is indicated by `bbox3d_fields`.

Reply:
Yep, I have also considered that case. I just wonder whether to unify the format of `bbox3d_fields` in [data loading](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/tutorials/data_pipeline.md#data-loading) and [pre-processing](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/tutorials/data_pipeline.md#pre-processing), or just to follow the reasonable pipeline in practice.

Reply:
I got your point. But at least `bbox3d_fields` itself is not changed right? I prefer leaving `*bbox3d_fields` unchanged here.

Reply:
You are definitely right. Maybe we need a further review.

Reply:
I think keep * will be better.